docs/references/nextjs/custom-signup-signin-pages.mdx

```diff
@@ -75,7 +75,7 @@ If Clerk's prebuilt components don't meet your specific needs or if you require
   ```tsx {{ prettier: false, filename: 'middleware.ts' }}
   import { clerkMiddleware } from "@clerk/nextjs/server";
 
-  const isPublicRoute = createRouteMatcher(["/signin(.*)", "/signup(.*)"]);
+  const isPublicRoute = createRouteMatcher(["/sign_in(.*)", "/sign_up(.*)"]);
```

Comment:
It looks like hyphen is the suggestion earlier in the guide, right?

```suggestion
  const isPublicRoute = createRouteMatcher(["/sign-in(.*)", "/sign-up(.*)"]);
```

Reply:
Mb, yes that is correct it should have been a hyphen